### ISSUE TITLE
Leaderboard cannot be hidden in multiplayer

### ISSUE THREAD

**Issue by Sclafus** [ppy/osu#27850]
### Type

Cosmetic

### Bug description

While playing in multiplayer lobbies the in-game leader board cannot be hidden by pressing tab. 
This works in solo.
This happens also when multiple people are in the lobby, of course.

### Screenshots or videos

https://github.com/ppy/osu/assets/31241607/9102e153-2843-4a2c-b0c9-964e7760d58d

### Version

2024.412.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14967896/compressed-logs.zip)

**Comment by smoogipoo** [comment]
This is intentional for the time being.

**Comment by Sclafus** [comment]
@smoogipoo can you please expand a bit?

**Comment by smoogipoo** [comment]
For lazer, we don't want to remove the "multiplayer" elements from multiplayer. That includes the leaderboard and the big `#X` text in the bottom right, if that ever returns.

**Comment by Sclafus** [comment]
> For lazer, we don't want to remove the "multiplayer" elements from multiplayer. That includes the leaderboard and the big `#X` text in the bottom right, if that ever returns.

While I understand where you are coming from, I feel like it does not take into account that a lot of players (especially during tournaments) would love to hide the leaderboard to concentrate more on the actual gameplay.
Also, the fact that pressing tab actually says "Enabled" and "Disabled" is quite missleading

**Comment by nixosenjoyer** [comment]
I agree. There should be an option to disable leaderboard during gameplay. For me personally it's very distracting, especially when player positions are constantly shuffling. I was also mislead with "Enabled" and "Disabled" popup and I thought this feature is just not working properly.

> For lazer, we don't want to remove the "multiplayer" elements from multiplayer

I don't think that leaderboard during gameplay is the big part of multiplayer. Also it could share the same logic as in solo gameplay so it can appear during break or player could check position by just pressing Z key.

I hope 🙏 you will rethink this decision, because some players (like me) can be very sensitive to movement on screen and can easily lose focus during gameplay.

**Comment by Zyfarok** [comment]
![IMG_20240803_170350](https://github.com/user-attachments/assets/9de82d4b-7d8c-47c2-b528-574da8bc77f8)

**Comment by Ladwick** [comment]
Hiding leaderboards during multiplayer is a must have for players that get nervous from seeing it during tournaments

**Comment by Nyxa** [comment]
Not just nerves. A moving leaderboard in the corner is murder for people with ADHD and other attention-deficit disabilities. Some people want to play maps with their friends, or play in tournaments, but have their performance (and experience) worsened with the constant movement going on in the leaderboards.

In a game that's as customizable as osu! (which is one of its big strengths) it makes no sense to me.

Furthermore, even if this were the case, having a UI element be visible when the toggle for disabling UI is on makes no sense to me either, and no one I've spoken to in multiplayer likes this. A lot of people think it's a bug that will be fixed in the future, not an intended feature.

**Comment by grovygay** [comment]
As a competitive player with many tournaments played. I do think you guys have made something great with lazer and it could very well replace stable in the future as the main client.

But to me, and many others I know, not being able to hide a leaderboard which depending on the closeness of the match can move constantly is pretty much a killer. At the level on which competitive osu! is played at, an unwarranted distraction is something so many of us simply can't afford.

I would please ask you to reconsider this design choice, or at least allow us to "hide" the element via skinning as we could do in stable before.

**Comment by bdach** [comment]
I'm tired of seeing this brought up everywhere so I'm authoritatively reopening.

That said, one caveat (taken from stream conversation):

![image](https://github.com/user-attachments/assets/9307282d-34d8-4f25-8c2f-307528f30964)
